website/source/docs/connect/ca/aws.html.md

```diff
@@ -0,0 +1,146 @@
+are used which means that suitable credentials and region configuration need to be present in one of:
+ 1. Environment variables
+ 2. Shared credentials file
+ 3. Via an EC2 instance role
+
+The IAM credential provided must have permission for the following actions:
+
+ - CreateCertificateAuthority - assuming an existing CA is not specified in `existing_arn`
+ - DescribeCertificateAuthority
+ - GetCertificate
+ - IssueCertificate
+
+## Configuration
+
+The ACM Private CA provider is enabled by setting the `ca_provider` to
+`"aws-pca"`. At this time there is only one, optional configuration value.
+
+```hcl
+connect {
+    enabled = true
+    ca_provider = "aws-pca"
+    ca_config {
+      existing_arn = "..."
+    }
+}
+```
+
+ ~> Note that suitable AWS IAM credentials are necessary for the provider to
+ work, however these are not configured in the Consul config which is typically
+ on disk and rely on the [standard AWS SDK configuration
+ locations](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials).
+
+The set of configuration options is listed below. The
+first key is the value used in API calls while the second key (after the `/`)
+is used if configuring in an agent configuration file.
+
+  * `ExistingARN` / `existing_arn` (`string: <optional>`) - The Amazon Resource
+    Name (ARN) of an existing private CA in your ACM account. If specified,
+    Consul will attempt to use the existing CA to issue certificates. Currently
+    this **must be a root CA** in the primary datacenter although in the future
+    we intend to support subordinate CAs. In a secondary datacenter, it must be
+    a subordinate CA who's root is the same as the root used in the primary
+    datacenter - if it's not Consul will create a new one from the Primary
+    datacenter's root.
+
+      The default behavior if this is not specified is for Consul to create a
+      new Root CA in the primary datacenter and a subordinate CA in each
+      secondary DC that it will automatically get signed by the primary's root.
+
+## Limitations
+
+ACM Private CA has several
+[limits](https://docs.aws.amazon.com/acm-pca/latest/userguide/PcaLimits.html)
+that restrict how fast certificates can be issued. This may impact how quickly
+large clusters can rotate all issued certificates.
+
+At the current time the ACM Private CA provider for Connect has some additional
+limitations described below.
+
+### Unable to Cross-sign Other CAs
+
+It's currently not possible to cross-sign other CA provider's root certificates
+during a migration. This is due to a ACM Private CA not having a mechanism to
+blindly cross-sign another root certificate without a CSR being generated. Both
+Consul's built in CA and Vault can do this and the current workflow for managing
+CAs relies on it.
+
+In the future it should be possible to remove this limitation by changing the
```

Comment:
I'm alway hesitant to talk about future functionality, unless it's not in-progress work, since the future could be 2 months or 2 years.

Reply:
That's fair. I'll remove those comments.